### PR TITLE
Rewrite and add to Smallest Common Multiple

### PR DESCRIPTION
**Pull request by erictleung**
Closes FreeCodeCamp/FreeCodeCamp#3410.

- Lints fenced code blocks
- Adds to beginning explanation section
- Add backticks to indicate code when appropriate
- Made text width 80 characters

Any feedback is welcome. I wasn't really sure what was expected for "rewrite and simplify explanation" of the challenge.

### CODE REVIEW

File: Algorithm-Smallest-Common-Multiple.md [hunk 1]
```diff
@@ -1,14 +1,34 @@
...
+all of them.
 
 ## Hint: 1
-Create an array with all the numbers that are missing from the original array to make it easier to check when having to check for even division.
+Create an array with all the numbers that are missing from the original array to
+make it easier to check when having to check for even division.
 
 ## Hint: 2
-You can use modulo `%` to check if the reminder is 0, which means it is evenly divisible.
+You can use modulo operator (`%`) to check if the reminder is 0, which means it is evenly divisible.
```
Comment (SaintPeter) [inline Algorithm-Smallest-Common-Multiple.md]: Technically this is the `remainder` operator - JavaScript does not have a `modulo` operator.  The main difference is what happens with a negative number:
http://www.sitecrafting.com/blog/modulus-remainder/

File: Algorithm-Smallest-Common-Multiple.md [hunk 2]
```diff
@@ -1,14 +1,34 @@
...
 
 ## Hint: 3
-If you sort the array from greater to lowest then you can check for the first two numbers as it is more likely to the the smallest common multiple than the lower numbers.
+If you sort the array from greater to lowest then you can check for the first
+two numbers as it is more likely to the smallest common multiple than the lower
+numbers.
```
Comment (SaintPeter) [inline Algorithm-Smallest-Common-Multiple.md]: Grammar: 
> two number as \[They\] are more likely to \[be\] the smallest common

File: Algorithm-Smallest-Common-Multiple.md [hunk 3]
```diff
@@ -53,58 +74,81 @@ function smallestCommons(arr) {
...
-- In the **do** part, we are going to multiply the very first number, times the number of loops, times the second number.
-- The loop part will allows us to increase the number beyond the greatest number we have without having to change the algorithm.
-- We enter a for loop that will go from n being 2 and going up by one while it is smaller than the array with all the numbers.
-- If the quotient is not even then stop the loop. If it is even then it check for the next elements in the array until it is not even or we find our answer.
-- Outside the loop, increase the value of loop.
-- At the end of the loop return the quotient.
-
-If the array only has two elements then the for loop never gets used and the return value is the product of said numbers. Otherwise, from the third element and until n is the same and the array length check if the remainder of the quotient and the third value of the array is not 0, if it is not 0 then stop loop increases and then we start over. If the remainder was 0 then keep checking until the end of the array.
+- Because the possibility of the smallest common denominator being among the two
```
Comment (SaintPeter) [inline Algorithm-Smallest-Common-Multiple.md]: > because \[of\] the possibility

**Comment by SaintPeter** [comment]
Some good changes here, just needs some minor edits.

**Comment by erictleung** [comment]
Make the edits! I also changed Hint 3 a little bit more too to hopefully make it easier to understand.

**Comment by SaintPeter** [comment]
Great work, thanks!

**Pull request merged**